### PR TITLE
OSDOCS-1351 Separate images command and output blocks

### PR DESCRIPTION
**Pull request by bmcelvee**
https://issues.redhat.com/browse/OSDOCS-1351

### CODE REVIEW

**Comment by bmcelvee** [comment]
@openshift/team-documentation please peer review, thanks!

File: modules/templates-rails-creating-frontend-service.adoc [hunk 1]
```diff
@@ -73,12 +78,14 @@ by running the database migration script. There are two ways you can do this:
 +
 ** Exec into frontend container with `rsh` command:
 +
+[source,terminal]
 ----
   $ oc rsh <FRONTEND_POD_ID>
```
Comment (bergerhoffer) [inline modules/templates-rails-creating-frontend-service.adoc:83]: There is some indentation that should be removed here, and can you change the replaceable value from <FRONTEND_POD_ID> to <frontend_pod_id> please?

The below code block is indented too.

File: modules/templates-rails-creating-welcome-page.adoc [hunk 2]
```diff
@@ -28,12 +29,14 @@ All the necessary files are created.
 
 . edit line 2 in `config/routes.rb` file as follows:
 +
+[source,terminal]
```
Comment (bergerhoffer) [inline modules/templates-rails-creating-welcome-page.adoc:32]: Should this really be source,terminal if it's a ruby file?

File: modules/templates-rails-storing-application-in-git.adoc [hunk 3]
```diff
@@ -39,10 +45,18 @@ vendor
...
++
+[source,terminal]
 ----
 $ git init
+----
++
+[source,terminal]
+----
 $ git add .
++
```
Comment (bergerhoffer) [inline modules/templates-rails-storing-application-in-git.adoc:57]: This is missing the ending `----`, so the rest of the module is messed up

File: modules/templates-rails-writing-application.adoc [hunk 4]
```diff
@@ -11,34 +11,44 @@ Rails gem first. Then you can proceed with writing your application.
 
 . Install the Rails gem:
 +
+[source,terminal]
 ----
 $ gem install rails
+----
+
```
Comment (bergerhoffer) [inline modules/templates-rails-writing-application.adoc:18]: This is missing a +, so the indentation and numbering is messed up

**Review by bergerhoffer — COMMENTED** [review]
A few things

**Comment by bmcelvee** [comment]
@bergerhoffer thank you!!

File: modules/images-allow-pods-to-reference-images-across-projects.adoc [hunk 5]
```diff
@@ -13,6 +13,7 @@ images in *project-b*, a service account in *project-a* must be bound to the
 . To allow Pods in *project-a* to reference images in *project-b*, bind a service
```
[the remark this answers is not in the document]
Comment (bmcelvee) [reply modules/images-allow-pods-to-reference-images-across-projects.adoc:13]: Sure thing!

File: modules/images-configuration-insecure.adoc [hunk 6]
```diff
@@ -16,6 +16,7 @@ only support HTTP connections, should be avoided.
 
 . Edit the `image.config.openshift.io/cluster` custom resource:
 +
+[source,terminal]
 ----
 $ oc edit image.config.openshift.io/cluster
 ----
```
[the remark this answers is not in the document]
Comment (bmcelvee) [reply modules/images-configuration-insecure.adoc:22]: Thank you! Updating - I did too quick of a pass on these files. -_-

**Pull request merged**